src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -631,10 +641,11 @@ def _check_pid_from_log_file(self, pid, abort_on_warning,
         total_lines = trace_lines + non_trace_lines
         p_trace = trace_lines * 1.0 / total_lines * 100
 
-        print("Pid {}, {} lines total, {} trace ({:.2f}%)".format(pid,
-                                                                  total_lines,
-                                                                  trace_lines,
-                                                                  p_trace))
+        if not self.quiet:
+            print("Pid {}, {} lines total, {} trace ({:.2f}%)".format(pid,
+                                                                      total_lines,
+                                                                      trace_lines,
+                                                                      p_trace))
```

Comment:
To avoid the line too-long pylint error you could use a [hanging indent](https://www.python.org/dev/peps/pep-0008/#indentation):
```suggestion
        if not self.quiet:
            print("Pid {}, {} lines total, {} trace ({:.2f}%)".format(
                pid, total_lines, trace_lines, p_trace))
```

Reply:
True, although I don't really think that's much easier to read.  Short line lengths don't really work with python string formatting, but I'll add this change next time I have to rebuild.